gensim/utils.py

```diff
@@ -2038,4 +2038,4 @@ def save_as_line_sentence(corpus, filename):
     """
     with smart_open(filename, mode='w', encoding='utf8') as fout:
         for sentence in corpus:
-            fout.write(' '.join(sentence) + '\n')
+            fout.write(u(' '.join(sentence) + '\n'))
```

Comment:
This won't work in Python 2. Please open the file in binary mode `wb` and always write out bytes (not text / unicode).

Also, the brackets are weird. Unicode literal is simply `u' '`.

Reply:
It was not unicode literal, it was `six.u` function, actually.